### PR TITLE
Adds the Elite M41A rifle and AP and incen mags to req.

### PR DESCRIPTION
**Pull request by Naaanii**
## About The Pull Request

Adds the Elite version of the M41A rifle to req, along with the AP and incen mags.

## Why It's Good For The Game

Adds an underused gun and unused ammo types, adding more variety to imports.

## Changelog
:cl:
add: Added the Elite M41A rifle to req
add: Added AP and Incen M41A mags to req
/:cl:

### CODE REVIEW

File: code/modules/reqs/supplypacks.dm [hunk 1]
```diff
@@ -1293,11 +1293,26 @@ Imports
...
+	contains = list(/obj/item/weapon/gun/rifle/m412/elite)
+	cost = 60
+
 /datum/supply_packs/imports/m41a2/ammo
 	name = "M412 Pulse Rifle Ammo"
 	contains = list(/obj/item/ammo_magazine/rifle)
 	cost = 5
 
+/datum/supply_packs/imports/m412/ammo/ap
+    name = "M412 AP Magazine"
```
Comment (Alterist) [inline code/modules/reqs/supplypacks.dm:1307]: Fucked up an indent here

Comment (Naaanii) [reply code/modules/reqs/supplypacks.dm:1307]: Odd, it was fine on my end.

**Comment by BraveMole** [comment]
Incends rounds are cancer to deal with as a xeno. Litteraly cancer

**Comment by TiviPlus** [comment]
no

**Pull request closed without merging**